sdk/src/feature_set.rs

```diff
@@ -773,6 +773,10 @@ pub mod enable_gossip_duplicate_proof_ingestion {
     solana_sdk::declare_id!("FNKCMBzYUdjhHyPdsKG2LSmdzH8TCHXn3ytj8RNBS4nG");
 }
 
+pub mod chained_merkle_conflict_duplicate_proofs {
+    solana_sdk::declare_id!("chaie9S2zVfuxJKNRGkyTDokLwWxx6kD2ZLsqQHaDD8");
```

Comment:
The risk of including the feature key here is that it will be misleading what minimum version is needed when deciding to activating the feature. ie. someone needs to recall what latest commit changed the associated logic.

Might be better to use a phone key for now and rekey once all the logic is in.

Reply:
good point, I put in a phoney key for now.
Will also create the feature gate issue for the final commit to make it clear.